manifests/common/plumgrid.pp

```diff
@@ -0,0 +1,16 @@
+# Common class used by Plumgrid Neutron Plugin.
+
+class openstack::common::plumgrid {
+  # forward all ipv4 traffic
+  # this is required for the vms to pass through the gateways public interface
+  sysctl::value { 'net.ipv4.ip_forward': value => '1' }
```

Comment:
This resource is already declared https://github.com/puppetlabs/puppetlabs-openstack/blob/master/manifests/profile/neutron/router.pp#L3 so this will cause a duplicate declaration error.